### PR TITLE
feat: add new page to redirect to when borrower says (from email)

### PR DESCRIPTION
**Pull request by yolile**
that they already started the lender onboarding process

related https://github.com/open-contracting/credere-backend/issues/424

### CODE REVIEW

**Review by jpmckinney — APPROVED** [review]
I trust, only had a quick look.

File: src/layout/PublicApplicationLayout.tsx [hunk 1]
```diff
@@ -41,7 +41,8 @@ export default function PublicApplicationLayout() {
         application.status === APPLICATION_STATUS.SUBMITTED ||
         application.status === APPLICATION_STATUS.STARTED
       ) {
-        if (lastSegment !== 'submission-completed') navigate('./submission-completed');
+        if (lastSegment === 'external-onboarding-completed') navigate('./external-onboarding-completed');
```
Comment (jpmckinney) [inline src/layout/PublicApplicationLayout.tsx:44]: When `lastSegment ===`, shouldn't it just return (like for `change-primary-email`)? Won't this navigate in a loop?

File: src/pages/msme/ExternalOnboardingCompleted.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { useT } from '@transifex/react';
```
Comment (jpmckinney) [inline src/pages/msme/ExternalOnboardingCompleted.tsx:1]: How do borrowers reach this page? I don't know what the flow is meant to be.

File: src/pages/msme/SubmissionCompleted.tsx [hunk 3]
```diff
@@ -27,7 +27,8 @@ function SubmissionCompleted() {
               },
             )}
           </Text>
-          {applicationContext.state.data?.lender.external_onboarding_url ? (
+          {applicationContext.state.data?.lender.external_onboarding_url &&
+          !applicationContext.state.data?.application.borrower_accessed_external_onboarding_at ? (
```
Comment (jpmckinney) [inline src/pages/msme/SubmissionCompleted.tsx:31]: In what scenario could a borrower reach this page after accessing the onboarding URL?

**Review by jpmckinney — COMMENTED** [review]
Actually I thought of a few questions.

File: src/pages/msme/ExternalOnboardingCompleted.tsx (hunk 2 above)
Comment (yolile) [reply src/pages/msme/ExternalOnboardingCompleted.tsx:1]: From the reminder email `EXTERNAL_ONBOARDING_DONE_URL`

File: src/pages/msme/SubmissionCompleted.tsx (hunk 3 above)
Comment (yolile) [reply src/pages/msme/SubmissionCompleted.tsx:31]: If they open their link again at any time

File: src/layout/PublicApplicationLayout.tsx (hunk 1 above)
Comment (yolile) [reply src/layout/PublicApplicationLayout.tsx]: Correct, that is better, changed it

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**